.github/workflows/main.yaml

```diff
@@ -105,7 +105,7 @@ jobs:
     - name: Upload coverage reports to Codecov with GitHub Action
       uses: codecov/codecov-action@v3
       with:
-        files: ./coverage1.txt,./coverage2.txt,./coverage3.txt,./addons/coverage.txt,./pinniped-components/post-deploy/coverage.txt,./pinniped-components/tanzu-auth-controller-manager/coverage.txt,./cli/core/coverage.txt,./cli/runtime/coverage.txt,./tkg/coverage.txt,./featuregates/client/cover.out,./featuregates/controller/cover.out,./capabilities/client/cover.out,./capabilities/controller/cover.out
+        files: ./coverage1.txt,./coverage2.txt,./coverage3.txt,./addons/coverage.txt,./pinniped-components/post-deploy/coverage.txt,./pinniped-components/tanzu-auth-controller-manager/coverage.txt,./cli/core/coverage.txt,./cli/runtime/coverage.txt,./tkg/coverage.txt,./featuregates/client/cover.out,./featuregates/controller/cover.out,./capabilities/client/cover.out,./capabilities/controller/cover.out./packages/cluster-api-provider-azure/test/coverage2.txt,./packages/cluster-api-provider-vsphere/test/coverage2.txt,./packages/cluster-api/test/coverage2.txt,./packages/cluster-api-provider-docker/test/coverage2.txt,./packages/cluster-api-bootstrap-kubeadm/test/coverage2.txt,./packages/cluster-api-provider-aws/test/coverage2.txt,
```

Comment:
Idea: can we standardize on a coverage file, have every component use that and then use a glob pattern to upload all of them?

Reply:
> standardize on a coverage file name

Good idea, but standardization at this point would take a lot of time, and I believe it would be wasted time given we're the changes we're not done with refactoring. Let's consider a standard and document our new rules once refactoring is complete and changes have settled down.